.github/workflows/coverage.yml

```diff
@@ -36,13 +63,9 @@ jobs:
         if: steps.grcov-cache.outputs.cache-hit != 'true'
         run: |
           cargo +stable install grcov
-  
-      - uses:                   actions-rs/toolchain@v1
-        with:
-          toolchain:            nightly-2022-12-26
-          override:             true
 
-      - run: cargo test --all-features --no-fail-fast
+      # Actual job
+      - run: cargo +${ICU4X_NIGHTLY_TOOLCHAIN} test --all-features --no-fail-fast
```

Comment:
`ICU4X_NIGHTLY_TOOLCHAIN` is not defined, and I think shouldn't be, because you have the toolchain override step